mne/io/artemis123/artemis123.py

```diff
@@ -32,6 +35,12 @@ def read_raw_artemis123(input_fname, preload=False, verbose=None):
         on the hard drive (slower, requires less memory).
     verbose : bool, str, int, or None
         If not None, override default verbose level (see mne.verbose).
+    pos_fname : str or None (defulat None)
+        If not None, load digitized head points from this file
+    head_loc : bool (default False)
+        If True attempt to perform head localization using HPI coils. If no
+        HPI coils are in info['dig'] hpic coils are assumed to be in canonical
+        order of fiducial points (nas, rpa, lpa).
```

Comment:
... and I'm not quite sure what this parameter does. Does it do continuous head localization? If so that should really be done by a call to `_calculate_chpi_positions`. If it computes `info['dev_head_t']` then it should explicitly mention computing the initial device to head coordinate transform.